.travis.yml

```diff
@@ -34,6 +34,7 @@ matrix:
         - sudo ssh default sudo podman build -t test /vagrant
         # Mounting /lib/modules into the container is necessary as CRIU wants to load (via iptables) additional modules
         - sudo ssh default sudo podman run --privileged --cgroupns=private -v /lib/modules:/lib/modules:ro test make localunittest
+        - sudo ssh default 'cd /vagrant && sudo ./script/install-bats.sh && make RUNC_USE_SYSTEMD=yes integration'
```

Comment:
* bats should be installed in Vagrantfile
* integration -> localintegration

Reply:
> bats should be installed in Vagrantfile

I did that initially, but by copy-pasting the installation steps from the Dockerfile. I'd rather have it in a script executed from both places. Is there a way to source the script to Vagrantfile (like I do for Dockerfile)?

Reply:
https://www.vagrantup.com/docs/provisioning/shell.html#external-script

Reply:
I guess, it should be also possible to invoke scripts under `/vagrant` from the inline script, but haven't confirmed.

Reply:
Seems like I found a way. Let me test locally and then I'll push it.